### DISCUSSION TITLE
more than one silo worker

### DISCUSSION THREAD

**Discussion by Jaguar880** [Courseplay/Courseplay_FS22#2920] [Ideas]
Hi!

Is it possible to use more than one bunker silo worker for levelling chaff in a silo? (might not be aware of it)
In real life, there is often more than one tractor levelling the silo.

Maybe this can be integrated in courseplay?
e.g. two tractors have a course each and recognise the other one when it comes too close, so that they backup.

Thanks for your great work! :)

**Comment by Tensuko** [comment]
Thats not possible or only with high effort.
Would need to split the bunker in 2 and that only make sense for large bunker. To much work for to less possible use cases.
